src/firebase/CRUDServices/imageService.js

```diff
@@ -28,68 +28,39 @@ export default class ImageService {
      * @param {String} userId user id.
      * @param {String} experimentId experiment id.
      * @param {File} image self image file.
-     * @return {Boolean} succeed or not.
+     * @return {String} succeed or not.
      */
-    postImage = async (userId, experimentId, image) => {
+    postRawImage = async (userId, experimentId, image) => {
       const folderPath = userId + '-' + experimentId + '/';
       const imagePath = folderPath + image.name;
-      const rawImageRef = self.sieRawImagesBucketRef.child(imagePath);
-      rawImageRef.put(image).then(() => {
-        return true;
+      const rawImageRef = self.storageRef.child(imagePath);
+      return await rawImageRef.put(image).then(() => {
+        return 'uploaded';
       }).catch((error) => {
-        return false;
+        return 'error';
```

Comment:
I'm not sure what to do here but I'm sure we should not do this

Reply:
Oh this is for testing for that day actually, so that the frontend team can use window.alert() to print the message on screen when uploaded the photo. Will discuss the proper way to handle this in the meeting.